Documentation/Development.md

```diff
@@ -40,20 +106,21 @@ situations where you may need to build X10 yourself.  These include:
 2. You are attempting to make changes to the X10 implementation itself, and must
 therefore build a new version.
 
-#### Requirements
+##### Requirements
 
 * Bazel. This can be installed [manually][bazel] or with
 [Bazelisk][bazelisk]. You will need a version supported by TensorFlow
 (between `_TF_MIN_BAZEL_VERSION` and `_TF_MAX_BAZEL_VERSION` as specified in
 [tensorflow/configure.py][configure.py]).
 * Python3 with [numpy][numpy].
 
-#### Building
+##### Building
```

Comment:
Reading over this, I feel like (R) 119-127 should be extracted out.  It is describing what X10 is rather than how to build it.  With the new structure, this seems like it belongs in the new `Components` section.  WDYT?

Reply:
agreed!